dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -181,6 +184,7 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink
   bool associated(const GUID_t& local, const GUID_t& remote,
                   bool local_reliable, bool remote_reliable,
                   bool local_durable, bool remote_durable,
+                  const RTPS::VendorId_t vendor_id,
```

Comment:
```suggestion
                  RTPS::VendorId_t vendor_id,
```

Or pass by `const &`

Reply:
I think I used const ref everywhere else so I wonder why I didn't here.